babel/messages/frontend.py

```diff
@@ -118,6 +118,12 @@ def finalize_options(self):
                                        'or the base directory')
 
     def run(self):
+        domains = self.domain.split()
+
+        for domain in domains:
+            self.__run_domain(domain)
+
+    def __run_domain(self, domain):
```

Comment:
What's the reason for that? I learned, double-underscore-prefix automatically adds the classname as prefix, so you have a kind of "private" member.